### PR TITLE
Update o365-data-locations.md

### PR DESCRIPTION
**Pull request by msbemba**
made changes to the OST Link

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7018

File: microsoft-365/enterprise/o365-data-locations.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ If Customer’s billing address is outside Europe and Customer has an Office 365
 
 ### How does Microsoft define data?
 
-Review our [definitions for different types of data](https://go.microsoft.com/fwlink/p/?linkid=864390) on the Microsoft Trust Center. In the [Microsoft Online Services Terms](https://go.microsoft.com/fwlink/p/?linkid=862403), Microsoft makes contractual commitments regarding customer data/your data. We refer to core customer data as the customer data that is committed to be stored at rest only within your region according to the [Microsoft Online Services Terms](https://go.microsoft.com/fwlink/p/?linkid=862403).
+Review our [definitions for different types of data](https://go.microsoft.com/fwlink/p/?linkid=864390) on the Microsoft Trust Center. In the [Microsoft Online Services Terms](https://www.microsoft.com/licensing/terms/product/PrivacyandSecurityTerms/all), Microsoft makes contractual commitments regarding customer data/your data. We refer to core customer data as the customer data that is committed to be stored at rest only within your region according to the [Microsoft Online Services Terms](https://www.microsoft.com/licensing/terms/product/PrivacyandSecurityTerms/all).
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/o365-data-locations.md]:
```suggestion
Review our [definitions for different types of data](https://go.microsoft.com/fwlink/p/?linkid=864390) on the Microsoft Trust Center. In the [Privacy & Security Terms](https://www.microsoft.com/licensing/terms/product/PrivacyandSecurityTerms/all), Microsoft makes contractual commitments regarding customer data/your data. We refer to core customer data as the customer data that is committed to be stored at rest only within your region according to the [Privacy & Security Terms](https://www.microsoft.com/licensing/terms/product/PrivacyandSecurityTerms/all).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dmwmsft Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@dmwmsft Kindly merge this PR. Thanks

**Review by dmwmsft — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thnaks

**Pull request merged**